toc/working-groups/foundational-infrastructure.md

```diff
@@ -142,6 +142,9 @@ areas:
   - name: tas-operability-bot
     github: tas-operability-bot
 - name: Identity and Auth (UAA)
+  members:
+  - name: Praveen K Kumar
+    github: praveenkalluri18
```

Comment:
Hi @praveenkalluri18,

thank you for this pr! We don't support the `member` role in the WGs definition. If you would like to be added as reviewer to the UAA are please move this after the `approvers` section an start a `reviewers` section like in https://github.com/cloudfoundry/community/blob/3e9889fc3835b5457b325656c4e59c7d196d3162/toc/working-groups/foundational-infrastructure.md?plain=1#L67-L69